### PR TITLE
Info unit group updates for org chart styling

### PR DESCRIPTION
**Pull request by csebianlander**
We are performing several actions manually for the Bureau Structure page, or for planned updates to that page, that could be performed in code:

1. Info unit item headings do not allow for H5s, only H2 through H4
2. Images in this section have a circular border that is applied via Photoshop, but could be applied easily with `border-radius` styling

This PR implements updates to the info unit group to allow for both of these to be performed in Wagtail and provides a bit of extra flexibility for potential expanded future use.

---

## Additions

- Ability to select H5 in "Level" dropdown of an info unit item or group
- New dropdown for "Border radius for images?" to choose either circular or rounded-corner styling for images
- New lines in CSS to support the rounded-corner styling

## How to test this PR

1. localhost
2. Login to admin panel
3. Find a page with info unit groups (e.g. /about-us/the-bureau/bureau-structure/operations/)
4. Test setting Level option to "H5" and setting "Border radius for images?" to either "Rounded corners" or "Circle"

## Screenshots
Screen Shot 2022-09-09 at 1 36 27 PM
Screen Shot 2022-09-09 at 1 36 20 PM
Screen Shot 2022-09-09 at 1 35 56 PM

## Notes and todos

- I wanted the new styling options to exist within `info-unit.less` for some structural consistency but could not get them to work there, so they instead live in the otherwise placeholder-y `images.less`. 
- There is probably some other code cleanup possible here!

## Checklist

- [ ] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [ ] Future todos are captured in comments and/or tickets
- [ ] Project documentation has been updated, potentially one or more of:
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

#### Browser testing

Visually tested in the following supported browsers:
- [ ] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Edge 18 (the last Edge prior to it switching to Chromium)
- [ ] Internet Explorer 11 and 8 (via emulation in 11's dev tools)
- [ ] Safari on iOS
- [ ] Chrome on Android

#### Accessibility

- [ ] Keyboard friendly (navigable with tab, space, enter, arrow keys, etc.)
- [ ] Screen reader friendly
- [ ] Does not introduce new errors or warnings in [WAVE](https://wave.webaim.org/extension/)

#### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Does not introduce new lint warnings
- [ ] Flexible from small to large screens

### CODE REVIEW

**Comment by chosak** [comment]
Pinging @anselmbradford @brandon-rusk for some FEWD eyes on the CSS changes.

File: cfgov/v1/blocks.py [hunk 1]
```diff
@@ -51,6 +51,7 @@ class HeadingLevelBlock(blocks.ChoiceBlock):
         ("h2", "H2"),
         ("h3", "H3"),
         ("h4", "H4"),
+        ("h5", "H5"),
```
Comment (chosak) [inline cfgov/v1/blocks.py:54]: I think this change will require a new Django migration. You'll need to run something like

```
cfgov/manage.py makemigrations v1 --name heading_block_h5s
```

to generate the migration file.

Comment (csebianlander) [reply cfgov/v1/blocks.py:54]: done!

**Review by brandon-rusk — CHANGES_REQUESTED** [review]
Overall looks good, just minor changes:

- Our standard is BEM and these CSS names should be prefixed with `m-` to represent a modifier. So I believe in this case it'd be `m-image_border_circle` and `m-image_border_round`. @anselmbradford can you confirm this or correct me if I'm wrong?

- There should be a new line after the final bracket in the CSS file `images.less`

**Comment by anselmbradford** [comment]
> Our standard is BEM and I believe these CSS names should be prefixed with m- to represent a modifier. So in this case it'd be m-image_border_circle and m-image_border_round. @anselmbradford can you confirm this or correct me if I'm wrong?

`m-` stands for "molecule" from [our atomic design period](https://github.com/cfpb/consumerfinance.gov/blob/7483ef5f7f670e75c745bda2306c24e6d1aa0a5c/docs/atomic-structure.md). A modifier would be `__modifier`.

Our BEM (block, element, modifier) is `block_element__modifier`, or if there are multiple words,`some-block_some-element__a-modifier`. A modifier works on an existing class, so ideally if a modifier is used, the classes would be `block_element block_element__modifier`, where the base class sets some styles and the modifier adds to those base styles.

We also have precedent for utilities, which don't fit nicely as an atomic block and are just adding a specific visual feature, for those we use the `u-` prefix. It looks to me like these class names would be utilities. 

Therefore, the names would likely be:

```
u-image_border-circle
u-image_border-rounded
```

Note that I wrote "rounded", not "round", for the second one, as "round" and "circle" sound equivalent. What the second style is doing is rounding the block, not making it completely round. Therefore "rounded".

**Comment by brandon-rusk** [comment]
> > Our standard is BEM and I believe these CSS names should be prefixed with m- to represent a modifier. So in this case it'd be m-image_border_circle and m-image_border_round. @anselmbradford can you confirm this or correct me if I'm wrong?
> 
> `m-` stands for "molecule" from [our atomic design period](https://github.com/cfpb/consumerfinance.gov/blob/7483ef5f7f670e75c745bda2306c24e6d1aa0a5c/docs/atomic-structure.md). A modifier would be `__modifier`.
> 
> Our BEM (block, element, modifier) is `block_element__modifier`, or if there are multiple words,`some-block_some-element__a-modifier`. A modifier works on an existing class, so ideally if a modifier is used, the classes would be `block_element block_element__modifier`, where the base class sets some styles and the modifier adds to those base styles.
> 
> We also have precedent for utilities, which don't fit nicely as an atomic block and are just adding a specific visual feature, for those we use the `u-` prefix. It looks to me like these class names would be utilities.
> 
> Therefore, the names would likely be:
> 
> ```
> u-image_border-circle
> u-image_border-rounded
> ```
> 
> Note that I wrote "rounded", not "round", for the second one, as "round" and "circle" sound equivalent. What the second style is doing is rounding the block, not making it completely round. Therefore "rounded".

Thanks for the correction, Ans! I knew I could count on you haha

**Comment by csebianlander** [comment]
Thanks to both of you! I've modified the class names and made other tweaks per the recommendations; let me know if I've missed anything.

**Review by brandon-rusk — APPROVED** [review]

**Comment by chosak** [comment]
@csebianlander my suggestion [above](https://github.com/cfpb/consumerfinance.gov/pull/7235#discussion_r967330206) was incomplete. Because the heading block is used by StreamFields in multiple Django apps besides `v1`, migrations need to be created for those other apps as well. So instead of

```
cfgov/manage.py makemigrations v1 --name heading_block_h5s
```

which only creates migrations for app `v1`, we instead need to do

```
cfgov/manage.py makemigrations --name heading_block_h5s
```

That should create the migrations for the `filing_instruction_guide`, `form_explainer`, `paying_for_college`, and `regulations3k` apps, which are missing and causing the migrations check to fail above.

**Comment by csebianlander** [comment]
says it's still failing here on github, but the check passes on my local...

**Comment by csebianlander** [comment]
Looks good now! I think we're OK to approve/merge (I'm still not authorized to merge)

**Pull request merged**